test/module/shared_model/validators/block_validator_test.cpp

```diff
@@ -111,8 +110,7 @@ TEST_F(BlockValidatorTest, InvalidBlock) {
   auto invalid_block =
       generateBlock(txs, std::vector<shared_model::crypto::Hash>{});
 
-  auto validation_result = validator_.validate(invalid_block);
-  ASSERT_TRUE(validation_result.hasErrors());
+  ASSERT_TRUE(validator_.validate(invalid_block));
```

Comment:
if it has a built-in boolean conversion, it is easier to use. If you find this assertion somewhat counter-intuitive, it may be because we are not used to that `validate` returns error now.

Reply:
if you feel like that, let's think of a better name for `validate`

Reply:
Built-in boolean conversion is used before the output, the idea is not to lose the formatting of validator result.

Reply:
you are right, but when we expect or assert the result of boolean conversion of an optional to be `true` and when we see the error message, the optional does not contain a value, so it has nothing interesting to be printed. in the cases where we expect false, the comparison is already done with `boost::none`.